api/views/song.py

```diff
@@ -0,0 +1,13 @@
+from rest_framework import viewsets
+
+from api.models import Song
+from api.serializers.song import SongSerializer
+
+
+class SongViewSet(viewsets.ModelViewSet):
+    """
+    API endpoint that allows all standard interactions with Songs.
+    """
+
+    queryset = Song.objects.all()
+    serializer_class = SongSerializer
```

Comment:
This is the ViewSet, it describes what actions can be taken on the endpoint (or it would, except ModelViewSet just says "automatically give me all of the possible actions for this model"). We'll also need one of these for each of `song_entry` and `songbook`.